release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1145,6 +1145,50 @@ With this update, the Ingress Operator no longer adds the `clientca-configmap` f
 [id="ocp-4-15-openshift-cli-bug-fixes"]
 ==== OpenShift CLI (oc)
 
+[discrete]
+[id="ocp-4-15-openshift-oc-mirror-bug-fixes"]
+==== oc-mirror
+
+Prior to this release, if a user wanted to filter operator packages by channel, that is, `mirror.operators.catalog.packages.channels`, they were required to include the default channel for the package, even if they had no intention of using bundles from that channel.  Based on this information, if the `imageSetConfig` did not include the default channel for the package,  the resulting catalog is considered invalid.
+
+[WARNING]
+====
+Modifying the default channel should not be taken lightly as it might have an impact on the upgrade paths of the operator. Using `defaultChannel` is under the responsibility of the end user.
```

Comment:
```suggestion
Modifying the default channel should not be taken lightly as it could have an impact on the operator's upgrade paths. The use of `defaultChannel` is the responsibility of the end user.
```

Reply:
Ack